docs/fundamentals/networking/sockets/tcp.md

```diff
@@ -1,22 +1,35 @@
 ---
-title: Use TcpClient and TcpListener
-description: Learn how the TcpClient and TcpListener classes handle TCP communication in .NET.
+title: Use TcpClient and TcpListener in .NET
+description: Learn how to use the TcpClient class to create a socket to request and receive data using TCP in .NET.
 author: IEvangelist
 ms.author: dapine
 ms.date: 08/24/2022
-helpviewer_keywords:
-  - "requesting data from Internet, TCP"
-  - "receiving data, TCP"
-  - "TcpClient class, about TcpClient class"
-  - "data requests, TCP"
-  - "application protocols, TCP"
+helpviewer_keywords: 
+  - "protocols, TCP"
   - "network resources, TCP"
   - "sending data, TCP"
   - "TCP"
-  - "protocols, TCP"
+  - "TcpClient class, about TcpClient class"
+  - "application protocols, TCP"
+  - "receiving data, TCP"
+  - "TcpListener class, about TcpListener class"
+  - "Socket class, about Socket class"
+  - "data requests, TCP"
+  - "requesting data from Internet, TCP"
   - "Internet, TCP"
 ---
 
+# TCP Overview
+
+To use the Transmission Control Protocol (TCP) services in .NET, use the <xref:System.Net.Sockets.TcpClient> and <xref:System.Net.Sockets.TcpListener> classes. These protocol classes are built on top of the <xref:System.Net.Sockets.Socket?displayProperty=nameWithType> class and take care of the details of transferring data.
```

Comment:
This is not the only, and not necessarily the preferred way to work with TCP. We should rephrase this intro in a way that makes it clear that `TcpClient` and `TcpListener` are only helper classes for ease of use.
I would even rename the file from `tcp.md` to `tcp-classes.md`.